### PR TITLE
Release Russian pack 8.6 and fixes 8.0.2, 8.1.1, 8.2.1, 8.5.1

### PR DESCRIPTION
**Pull request by theshadowco**
New release and small fixes

### CODE REVIEW

File: l10nru.properties [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
 
+8.6.description=Support SonarQube 8.6
+8.6.date=2021-01-01
+8.6.sqVersions=[8.6,LATEST]
+8.6.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.6/sonar-l10n-ru-plugin-8.6.jar
+8.6.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.6
+
+8.2.1.description=Small translation fixes
+8.2.1.date=2021-01-01
+8.2.1.sqVersions=[8.2,8.2]
```
Comment (ganncamp) [inline l10nru.properties:18]: FYI, no need to use range syntax when there's no range. This could be just `8.2`

File: l10nru.properties [hunk 2]
```diff
@@ -1,15 +1,45 @@
...
+publicVersions=7.9.1,8.0.2,8.1.1,8.2.1,8.3.1,8.4.1,8.5.1,8.6
+archivedVersions=1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,1.10,8.0,8.3,8.4,8.5,8.0.1,8.1,8.2
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-l10n-ru-plugin
 
+8.6.description=Support SonarQube 8.6
+8.6.date=2021-01-01
+8.6.sqVersions=[8.6,LATEST]
+8.6.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.6/sonar-l10n-ru-plugin-8.6.jar
```
Comment (ganncamp) [inline l10nru.properties:13]: the artifact is 404

File: l10nru.properties [hunk 3]
```diff
@@ -1,15 +1,45 @@
...
+8.6.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.6
+
+8.2.1.description=Small translation fixes
+8.2.1.date=2021-01-01
+8.2.1.sqVersions=[8.2,8.2]
+8.2.1.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.2.1/sonar-l10n-ru-plugin-8.2.1.jar
```
Comment (ganncamp) [inline l10nru.properties:19]: 404

File: l10nru.properties [hunk 4]
```diff
@@ -1,15 +1,45 @@
...
+8.2.1.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.2.1
+
+8.1.1.description=Small translation fixes
+8.1.1.date=2021-01-01
+8.1.1.sqVersions=[8.1,8.1]
+8.1.1.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.1.1/sonar-l10n-ru-plugin-8.1.1.jar
```
Comment (ganncamp) [inline l10nru.properties:25]: 404

File: l10nru.properties [hunk 5]
```diff
@@ -1,15 +1,45 @@
...
+8.1.1.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.1.1
+
+8.0.2.description=Small translation fixes
+8.0.2.date=2021-01-01
+8.0.2.sqVersions=[8.0,8.0]
+8.0.2.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.0.2/sonar-l10n-ru-plugin-8.0.2.jar
```
Comment (ganncamp) [inline l10nru.properties:31]: 404

File: l10nru.properties [hunk 6]
```diff
@@ -1,15 +1,45 @@
...
+8.0.2.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.0.2
+
+8.5.1.description=Small translation fixes
+8.5.1.date=2021-01-01
+8.5.1.sqVersions=[8.5,8.5.*]
+8.5.1.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.5.1/sonar-l10n-ru-plugin-8.5.1.jar
```
Comment (ganncamp) [inline l10nru.properties:37]: This one's actually 200! Woo hoo!

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: l10nru.properties (hunk 2 above)
Comment (theshadowco) [reply l10nru.properties:13]: oops :-( fixed

File: l10nru.properties (hunk 3 above)
Comment (theshadowco) [reply l10nru.properties:19]: oops :-( fixed

File: l10nru.properties (hunk 4 above)
Comment (theshadowco) [reply l10nru.properties:25]: oops :-( fixed

File: l10nru.properties (hunk 5 above)
Comment (theshadowco) [reply l10nru.properties:31]: oops :-( fixed

File: l10nru.properties (hunk 6 above)
Comment (theshadowco) [reply l10nru.properties:37]: :-)

**Review by ganncamp — APPROVED** [review]

**Pull request merged**